extensions/binding/org.eclipse.smarthome.binding.hue/ESH-INF/thing/channels.xml

```diff
@@ -33,4 +33,25 @@
         <category>ColorLight</category>
     </channel-type>
 
+    <!-- Alert Channel -->
+    <channel-type id="alert">
+        <item-type>String</item-type>
+        <label>Alert</label>
+        <description>The alert channel allows a temporary change to the bulb’s state.</description>
+        <state readOnly="true">
+            <options>
+                <option value="NONE">Not performing an alert effect</option>
```

Comment:
the options should be short labels as they are displayed on selection boxes by the UI.
So I would suggest "None", "Alert" and "Long Alert".